recipes-bsp/u-boot/u-boot-toradex_2020.07.bb

```diff
@@ -28,10 +30,19 @@ do_configure[cleandirs] = "${B}"
 
 inherit fsl-u-boot-localversion
 
-# U-Boot is flashed 1k into a NAND block, create a binary which prepends
-# U-boot with 1k of zeros to ease flashing
+PADDING_DIR = "${B}"
```

Comment:
I'd just replace `${PADDING_DIR}` to `${B}` as this is standard and used for all intermediate building artifacts.

Reply:
We had a use case were we built multiple U-Boot binaries.

There we bbappend PADDING_DIR = "${B}/${the nand config}".
While we no longer use it, I'd like to keep this indirection.
What do you think?

Reply:
But this can be handled the same way we handle the uboot_config case. It allows the multiple binaries build as well.

Reply:
Ok. I drop it.